fastlane/Fastfile

```diff
@@ -1,3 +0,0 @@
-lane :test do
-  firebase_app_distribution
-end
```

Comment:
I think I remember when this and fastlane/Pluginfile were deleted, but I don't remember why. Is there a particular reason? If not I think we should restore them. Looks like it has something to do with running tests.